piperider_cli/cloud/__init__.py

```diff
@@ -234,9 +234,9 @@ def list_reports(self, project_id, datasource=None):
         if not self.available:
             self.raise_error()
 
-        url = self.service.url(f'/api/projects/{project_id}/reports')
+        url = self.service.url(f'/api/v2/projects/{project_id}/reports')
```

Comment:
migrate to run api

* `/workspaces/{workspace_name}/projects/{project_name}/runs`